doc/source/data/faq.rst

```diff
@@ -0,0 +1,336 @@
+* Ray Datasets is used as the data processing engine for the 
+  `Ray-based Apache Beam runner <https://github.com/ray-project/ray_beam_runner>`__.
+* Ray Datasets is used as the preprocessing and batch inference engine for
+  :ref:`Ray AIR <air>`.
+
+
+If you're using Ray Datasets, please let us know about your experience on the
+`Slack <https://forms.gle/9TSdDYUgxYs8SA9e8>`__  or
+`Discourse <https://discuss.ray.io/>`__; we'd love to hear from you!
+
+What should I use Ray Datasets for?
+===================================
+
+Ray Datasets is the standard way to load, process, and exchange data in Ray libraries
+and applications, with a particular emphasis on ease-of-use, performance, and
+scalability in both data size and cluster size. Within that, Datasets is designed for
+two core uses cases:
+
+* **ML (training) ingest:** Loading, preprocessing, and ingesting data into one or more
+  (possibly distributed) model trainers.
+* **Batch inference:** Loading, preprocessing, and performing parallel batch
+  inference on data.
+
+We have designed the Datasets APIs, data model, execution model, and
+integrations with these use cases in mind, and have captured these use cases in
+large-scale nightly tests to ensure that we're hitting our scalability, performance,
+and efficiency marks for these use cases.
+
+See our :ref:`ML preprocessing feature guide <datasets-ml-preprocessing>` for more
+information on this positioning.
+
+What should I not use Ray Datasets for?
+=======================================
+
+Ray Datasets is not meant to be used for generic ETL pipelines (like Spark) or
+scalable data science (like Dask, Modin, or Mars). However, each of these frameworks
+are :ref:`runnable on Ray <data_integrations>`, and Datasts integrates tightly with
+these frameworks, allowing for efficient exchange of distributed data partitions often
+with zero-copy. Check out the
+:ref:`dataset creation feature guide <dataset_from_in_memory_data_distributed>` to learn
+more about these integrations.
+
+Datasets is specifically targeting
+the ML ingest and batch inference use cases, with focus on data loading and last-mile
+preprocessing for ML pipelines. For more information on this distinction, what we
+mean by last-mile preprocessing, and how Ray Datasets fits into a larger ML pipeline
+picture, please see our :ref:`ML preprocessing feature guide <datasets-ml-preprocessing>`.
+
+For data loading for training, how does Ray Datasets compare to other solutions?
+================================================================================
+
+There are several ML framework-specific and general solutions for loading data into
+model trainers. Below, we summarize some advantages Datasets offers over these more
+specific ingest frameworks.
+
+Torch datasets (and data loaders)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* **Framework-agnostic:** Datasets is framework-agnostic and portable between different
```

Comment:
It looks we are mostly comparing the same set of dimensions. How about groupby(comp_dimension) and then comment on each, or even better put them in a table?

Reply:
I was going to put that off for the sake of not having to refactor this again, but I'll take a stab at it.

Reply:
I have a WIP table local but I won't get it done before my laptop dies, let's do this as a follow-up.